645b215a29f36ff7921f0180d9477a8a15df88b4
apollo-common/src/main/java/com/ctrip/framework/apollo/common/exception/NotFoundException.java
--- 
+++ 
@@ -1,30 +1,30 @@
 /*
- * Copyright 2022 Apollo Authors
+ * Copyright 2023 Apollo Authors
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
  */
 package com.ctrip.framework.apollo.common.exception;
 
 import org.springframework.http.HttpStatus;
 
 public class NotFoundException extends AbstractApolloHttpException {
 
   /**
    * @see AbstractApolloHttpException#AbstractApolloHttpException(String, Object...)
    */
   public NotFoundException(String msgTpl, Object... args) {
     super(msgTpl, args);
     setHttpStatus(HttpStatus.NOT_FOUND);
   }
 }